kernel/src/hil/time.rs

```diff
@@ -93,8 +113,33 @@ pub trait Client {
 /// The `Timer` trait models a timer that can notify when a particular interval
 /// has elapsed.
 pub trait Timer: Time {
+    fn set_client(&self, client: &'static TimerClient);
+
     /// Sets a one-shot timer to fire in `interval` clock-tics.
     fn oneshot(&self, interval: u32);
     /// Sets repeating timer to fire every `interval` clock-tics.
     fn repeat(&self, interval: u32);
+
```

Comment:
A second call to `oneshot` or `repeat` will effectively overwrite the first call, correct? I think this should be more clearly documented

Reply:
Yes, good point.